src/libsyntax/attr/builtin.rs

```diff
@@ -27,7 +27,7 @@ enum AttrError {
     UnsupportedLiteral
 }
 
-fn handle_errors(diag: &Handler, span: Span, error: AttrError) {
+fn handle_errors(diag: &Handler, span: Span, error: AttrError, is_bytestr: bool) {
```

Comment:
`is_bytestr` is only used with `AttrError::UnsupportedLiteral`, so it can be made it's part - `AttrError::UnsupportedLiteral(bool /* is_bytestr */)`